x/mint/README.md

```diff
@@ -1,11 +1,13 @@
 # Mint
 
```

Comment:
Should we start moving files for other modules too? 
like `x/mint/spec/README.md -> x/ming/README.md`

Reply:
https://github.com/osmosis-labs/osmosis/issues/1864